engine/vcs/gitlab/client_status.go

```diff
@@ -127,7 +140,7 @@ func processWorkflowNodeRunEvent(event sdk.Event, uiURL string) (statusData, err
 	data := statusData{}
 	var eventNR sdk.EventRunWorkflowNode
 	if err := mapstructure.Decode(event.Payload, &eventNR); err != nil {
-		return data, sdk.WrapError(err, "cannot read payload")
+		return data, sdk.WrapError(err, "Cannot read payload")
```

Comment:
We prefer message that starts lower case because they will be wrapped.

Reply:
Okay, as I saw others WrapError messages with an uppercase first letter, so I changed it to be conformed. I'll change them